container-action/README.md

```diff
@@ -0,0 +1,59 @@
+# Rootstock Integration Tests Action
+
+This action provides a containerized environment for running integration tests on Rootstock. 
+It receives as inputs the branches of `powpeg`, `rskj` and `rootstock-integration-tests` repositories,
+checkout at the branches passed as parameters, build the projects and run the integration tests.
+
+The rootstock-integration-tests it's a project that tests the integration between rskj and powpeg-node, 
+it validates that the peg-in and peg-out processes are working correctly. It's extremely important to both projects, 
+and should be executed before any release of both projects or any merge to the master/main branch. 
+
+To achieve this and make this test more accessible, we created a container-action created to execute this test, 
+it offers the flexibility to run the tests with any specific tag or branch from *powpeg-node* or *rskj*. 
+That way, we will add steps on each repository to run the integration tests with the version that we want to test. 
+No matter if it's a tag, a branch or a specific commit.
+
+## Inputs
+By default, all the inputs are pointed to the `master/main` branch of the repositories. But, ideally, we will adapt
+the action to receive the branches  received to be the one in a branch or tag that we want to test.
+
+### `rskj-branch`
+
+The rskj branch to checkout. If no branch or tag passed, it will be used the default `master`.
+
+### `powpeg-node-branch`
+
+The powpeg-node branch to checkout. If no branch or tag passed, it will be used the default `master`.
+
+### `rit-branch`
+
+**Optional** The rootstock-integration-tests branch to checkout. This one it's optional, because it will be
+very unlikely that we need to use a different branch for the rootstock-integration-test. It's offered the possibility
```

Comment:
I wouldn't say it's very unlikely. Sometimes a new feature requires a change in the RIT so we need to use a specific RIT branch to run the tests along with the feature changes in rskj/powpeg.

Also, when there is a change to the RIT code and a PR against master we also want the tests to run with the RIT branch that has the changes included